sno/commit.py

```diff
@@ -75,15 +95,32 @@ def commit(ctx, message, message_file, allow_empty):
     elif message:
         commit_msg = "\n\n".join([m.strip() for m in message]).strip()
     else:
-        commit_msg = get_commit_message(repo, wc_changes)
+        commit_msg = get_commit_message(repo, wc_changes, quiet=is_output_json)
 
     if not commit_msg:
         raise click.Abort()
 
-    new_commit = rs.commit(wcdiff, commit_msg, allow_empty=allow_empty)
+    rs.commit(wcdiff, commit_msg, allow_empty=allow_empty)
+    new_commit = repo.head.peel(pygit2.Commit)
+
+    branch = None if repo.head_is_detached else repo.branches[repo.head.shorthand].shorthand
+    jdict = {
+        "sno.commit/v1": {
```

Comment:
I think we should _try_ and mirror [git](https://git-scm.com/docs/gitglossary) and/or [libgit's](https://libgit2.org/libgit2/#HEAD) terminology here as much as possible.

(no particular comments here, I think these are all fine)